<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Reproducible Research | Portfolio Naresh</title>
  <meta name="description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Reproducible Research | Portfolio Naresh" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  <meta name="github-repo" content="NareshdeBruijn14/Naresh_portfolio" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Reproducible Research | Portfolio Naresh" />
  
  <meta name="twitter:description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  

<meta name="author" content="Naresh de Bruijn" />


<meta name="date" content="2024-05-26" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="id_04.html"/>
<link rel="next" href="id_06.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/core-js-2.5.3/shim.min.js"></script>
<script src="libs/react-18.2.0/react.min.js"></script>
<script src="libs/react-18.2.0/react-dom.min.js"></script>
<script src="libs/reactwidget-1.0.0/react-tools.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="libs/reactable-binding-0.4.4/reactable.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="layout.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="toc"><h2>Table of Contents</h2></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Home Page</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#introduction"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#table-of-contents"><i class="fa fa-check"></i>Table of Contents</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_00.html"><a href="id_00.html"><i class="fa fa-check"></i>Setup</a>
<ul>
<li class="chapter" data-level="" data-path="id_00.html"><a href="id_00.html#packages"><i class="fa fa-check"></i>Packages</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html"><i class="fa fa-check"></i>Resumé</a>
<ul>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#personal-details"><i class="fa fa-check"></i>Personal details</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#allow-me-to-introduce-myself-briefly"><i class="fa fa-check"></i>Allow me to introduce myself briefly</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#internship-major-project"><i class="fa fa-check"></i>Internship / (major) project</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#relevant-coursework"><i class="fa fa-check"></i>Relevant Coursework</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#experiences"><i class="fa fa-check"></i>Experiences</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html"><i class="fa fa-check"></i>Free schedule plan</a>
<ul>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#introduction-1"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#goals"><i class="fa fa-check"></i>Goals</a></li>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#plan"><i class="fa fa-check"></i>Plan</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_03.html"><a href="id_03.html"><i class="fa fa-check"></i>Guerrilla Analytics</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html"><i class="fa fa-check"></i>C.elegans plate experiment</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#introduction-2"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#import-and-inspect-dataset"><i class="fa fa-check"></i>Import and inspect dataset</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatter-plot"><i class="fa fa-check"></i>Scatter plot</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#inspection"><i class="fa fa-check"></i>Inspection</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatterplot-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Scatterplot CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#analysis-plan"><i class="fa fa-check"></i>Analysis plan</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#normalising-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Normalising CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatterplot-normalised-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Scatterplot normalised CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#conclusion"><i class="fa fa-check"></i>Conclusion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html"><i class="fa fa-check"></i>Reproducible Research</a>
<ul>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#open-peer-review"><i class="fa fa-check"></i>Open peer review</a>
<ul>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#article-overview"><i class="fa fa-check"></i>Article overview</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#peer-review"><i class="fa fa-check"></i>Peer review</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#conclusion-1"><i class="fa fa-check"></i>Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#reproducible-analysis"><i class="fa fa-check"></i>Reproducible analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_06.html"><a href="id_06.html"><i class="fa fa-check"></i>R package</a>
<ul>
<li class="chapter" data-level="" data-path="id_06.html"><a href="id_06.html#examples"><i class="fa fa-check"></i>Examples</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html"><i class="fa fa-check"></i>Projecticum: Pediatric Cancer</a>
<ul>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#introduction-4"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#cfdna"><i class="fa fa-check"></i>cfDNA</a></li>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#r-shiny-app"><i class="fa fa-check"></i>R shiny app</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html"><i class="fa fa-check"></i>Parameterisation of COVID-19 Datasets</a>
<ul>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#introduction-5"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#parameters"><i class="fa fa-check"></i>Parameters</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#visualisation"><i class="fa fa-check"></i>Visualisation</a>
<ul>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#covid-19-cases"><i class="fa fa-check"></i>Covid-19 cases</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#covid-19-deaths"><i class="fa fa-check"></i>Covid-19 deaths</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html"><i class="fa fa-check"></i>BS-Seq</a>
<ul>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#introduction-6"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#bisulfite-conversion"><i class="fa fa-check"></i>Bisulfite conversion</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#bismark"><i class="fa fa-check"></i>Bismark</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#data"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#genome-preparation"><i class="fa fa-check"></i>Genome preparation</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#alignment"><i class="fa fa-check"></i>Alignment</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#deduplication"><i class="fa fa-check"></i>Deduplication</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#methylation-extraction"><i class="fa fa-check"></i>Methylation extraction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_20.html"><a href="id_20.html"><i class="fa fa-check"></i>References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio Naresh</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class="hero-image"></div>

<link href="layout.css" rel="stylesheet">
<div class="hero-image-container">
  <img class= "hero-image" src="documents/images/DNA_helix_blue.png">
</div>

<div id="id_05" class="section level1 unnumbered">
<h1>Reproducible Research</h1>
<div id="open-peer-review" class="section level2 unnumbered">
<h2>Open peer review</h2>
<div id="introduction-3" class="section level3 unnumbered">
<h3>Introduction</h3>
<p class="justify0">
The COVID-19 pandemic has underscored the critical role of open science and reproducible research. Open science facilitates the free exchange of findings, while reproducible research ensures that publications are both accessible and verifiable, thereby enhancing transparency and trust in scientific conclusions. Together, these practices drive scientific progress forward <span class="citation">[<a href="#ref-sumnerReproducibilityReportingPractices2020">1</a>]</span>. The transparency criteria of <a href="https://www.researchgate.net/publication/340244621_Reproducibility_and_reporting_practices_in_COVID-19_preprint_manuscripts">Reproducibility and reporting practices in COVID-19 preprint manuscripts</a> was used to examine the reproducibility of an published article. I’ll peer review <a href="https://doi.org/10.1101/2024.04.30.591893">Applying Cell Painting in Non-Tumorigenic Breast Cells to Understand Impacts of Common Chemical Exposures</a> from <a href="https://www.biorxiv.org/">bioRxiv</a> using the transparency criteria shown in table 2.
</p>
</div>
<div id="article-overview" class="section level3 unnumbered">
<h3>Article overview</h3>
<p class="justify0">
Researchers used Cell Painting to study the effects of various chemicals on mammary cells to understand their potential links to breast cancer. They examined 16 chemicals and 21 small molecules, analysing over 3000 cellular features. By comparing their findings with human exposure data, they identified similarities between certain chemicals and known pathways. Notably, a pesticide metabolite showed similarities to a Wnt pathway activator, suggesting a potential mechanism for breast cancer risk. This study highlights Cell Painting’s utility in understanding the effects of common environmental toxins on breast cancer<span class="citation">[<a href="#ref-tapaswiApplyingCellPainting2024">2</a>]</span>.
</p>
</div>
<div id="peer-review" class="section level3 unnumbered">
<h3>Peer review</h3>
<p>Table 2. The Ripeta team analyzed eight reproducibility criteria <span class="citation">[<a href="#ref-sumnerReproducibilityReportingPractices2020">1</a>]</span>.</p>
<table style="width:100%;">
<colgroup>
<col width="18%" />
<col width="51%" />
<col width="13%" />
<col width="16%" />
</colgroup>
<thead>
<tr class="header">
<th><strong>Transparency Criteria</strong></th>
<th><strong>Definition</strong></th>
<th>REsponse Type</th>
<th>Score</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Study Purpose</td>
<td>A concise statement in the introduction of the article, often in the last paragraph, that establishes the reason the research was conducted. Also called the study objective.</td>
<td>Binary</td>
<td>yes</td>
</tr>
<tr class="even">
<td>Data Availability Statement</td>
<td>A statement, in an individual section offset from the main body of text, that explains how or if one can access a study’s data. The title of the section may vary, but it must explicitly mention data; it is therefore distinct from a supplementary materials section.</td>
<td>Binary</td>
<td>yes</td>
</tr>
<tr class="odd">
<td>Data Location</td>
<td>Where the article’s data can be accessed, either raw or processed.</td>
<td>Found Value</td>
<td>yes; all in paper or supplementary fils</td>
</tr>
<tr class="even">
<td>Study Location</td>
<td>Author has stated in the methods section where the study took place or the data’s country/region of origin.</td>
<td>Binary; Found Value</td>
<td>no</td>
</tr>
<tr class="odd">
<td>Author Review</td>
<td>The professionalism of the contact information that the author has provided in the manuscript.</td>
<td></td>
<td>Tier 3</td>
</tr>
<tr class="even">
<td>Ethics Statement</td>
<td>A statement within the manuscript indicating any ethical concerns, including the presence of sensitive data.</td>
<td>Binary</td>
<td>no</td>
</tr>
<tr class="odd">
<td>Funding Statement</td>
<td>A statement within the manuscript indicating whether or not the authors received funding for their research.</td>
<td>Binary</td>
<td>yes</td>
</tr>
<tr class="even">
<td>Code Availability</td>
<td>Authors have shared access to the most updated code that they used in their study, including code used for analysis.</td>
<td>Binary</td>
<td>no</td>
</tr>
</tbody>
</table>
</div>
<div id="conclusion-1" class="section level3 unnumbered">
<h3>Conclusion</h3>
<p class="justify0">
This article meets most criteria at a sufficient level. However, it falls short in a couple of key areas. Firstly, it does not specify the study locations in the methods section. Secondly, there is no code available for the analysis, hindering the replicability of the results. These shortcomings could impact the article’s reception in open peer review processes, highlighting areas for improvement in transparency and reproducibility. Furthermore, there was no ethics statement mentioned. This may not be relevant in this study, but it should be included in the ethical statement section. With these shortcomes, the article does not pass the open peer review.
</p>
<hr />
</div>
</div>
<div id="reproducible-analysis" class="section level2 unnumbered">
<h2>Reproducible analysis</h2>
<p class="justify0">
From <a href="https://www.r-bloggers.com/2022/03/newish-paper-share-the-code-not-just-the-data-a-case-study-of-the-reproducibility-of-jml-articles-published-under-the-open-data-policy/">this blog</a>, I chose a open source paper using the <a href="https://osf.io/?view_only=">OSF website</a> to reproduce a peace of code that is used in the study. From <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849410/">this paper</a>, I have reproduced a piece of code. The associated code and datasets can be found <a href="https://osf.io/v4se2/">here</a>.
</p>
<p class="justify0">
The codes in this RMarkdown were used to create the figures presented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849410/">this paper</a>. On a scale from 1 (very poor) to 5 (very good), I rate the readability of the codes in this file a 4. Below, I reproduce a portion of the data to recreate figure 1 from the paper. It was easily to reproduce this figure, with a difficulty rating of 5 on a scale from 1 (very difficult) to 5 (very easy). I had to make a few minor adjustments to create the figure due to the updated version of ggplot. Further, the figure that was generated with this code is identical to the one in the paper.
</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="id_05.html#cb29-1" tabindex="-1"></a><span class="co"># Replication Study 50</span></span>
<span id="cb29-2"><a href="id_05.html#cb29-2" tabindex="-1"></a><span class="co"># Figure 1</span></span>
<span id="cb29-3"><a href="id_05.html#cb29-3" tabindex="-1"></a><span class="co"># Relative abundance of Fusobacterium in CRC vs adjacent normal (combined independent replicates)</span></span>
<span id="cb29-4"><a href="id_05.html#cb29-4" tabindex="-1"></a><span class="co"># Samples with detectable Fusobacterium in CRC and adjacent normal</span></span>
<span id="cb29-5"><a href="id_05.html#cb29-5" tabindex="-1"></a><span class="co"># R Version 3.4.2</span></span>
<span id="cb29-6"><a href="id_05.html#cb29-6" tabindex="-1"></a></span>
<span id="cb29-7"><a href="id_05.html#cb29-7" tabindex="-1"></a><span class="co"># Required Packages</span></span>
<span id="cb29-8"><a href="id_05.html#cb29-8" tabindex="-1"></a><span class="fu">library</span>(httr) <span class="co"># version 1.2.1</span></span>
<span id="cb29-9"><a href="id_05.html#cb29-9" tabindex="-1"></a><span class="fu">library</span>(rjson) <span class="co">#version 0.2.15</span></span>
<span id="cb29-10"><a href="id_05.html#cb29-10" tabindex="-1"></a><span class="fu">library</span>(plotrix) <span class="co"># version 3.6-6</span></span>
<span id="cb29-11"><a href="id_05.html#cb29-11" tabindex="-1"></a><span class="fu">library</span>(ggplot2) <span class="co"># version 2.2.1</span></span>
<span id="cb29-12"><a href="id_05.html#cb29-12" tabindex="-1"></a><span class="fu">library</span>(cowplot) <span class="co">#version 0.8.0</span></span>
<span id="cb29-13"><a href="id_05.html#cb29-13" tabindex="-1"></a><span class="co">#source(&quot;~/credentials.R&quot;) #for private use during r script generation</span></span>
<span id="cb29-14"><a href="id_05.html#cb29-14" tabindex="-1"></a></span>
<span id="cb29-15"><a href="id_05.html#cb29-15" tabindex="-1"></a><span class="co">#Downloads R script &quot;download.OSF.file.R&quot;</span></span>
<span id="cb29-16"><a href="id_05.html#cb29-16" tabindex="-1"></a><span class="fu">GET</span>(<span class="st">&quot;https://osf.io/hkpjb/?action=download&quot;</span>, <span class="fu">write_disk</span>(<span class="st">&quot;download.OSF.file.R&quot;</span>, <span class="at">overwrite =</span> <span class="cn">TRUE</span>))</span></code></pre></div>
<pre><code>## Response [https://files.osf.io/v1/resources/p7ayb/providers/osfstorage/59a5b71f9ad5a10255d6f93e?action=download&amp;direct&amp;version=7]
##   Date: 2024-05-26 13:15
##   Status: 200
##   Content-Type: application/octet-stream
##   Size: 3.58 kB
## &lt;ON DISK&gt;  download.OSF.file.R</code></pre>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb31-1"><a href="id_05.html#cb31-1" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;download.OSF.file.R&quot;</span>)</span>
<span id="cb31-2"><a href="id_05.html#cb31-2" tabindex="-1"></a><span class="co">#calls the download.OSF.file</span></span>
<span id="cb31-3"><a href="id_05.html#cb31-3" tabindex="-1"></a></span>
<span id="cb31-4"><a href="id_05.html#cb31-4" tabindex="-1"></a><span class="co">#Downloads data file &#39;Study_50_combined_qPCR_statistical_analysis.R&#39; from https://osf.io/8vcea/</span></span>
<span id="cb31-5"><a href="id_05.html#cb31-5" tabindex="-1"></a><span class="fu">download.OSF.file</span>(<span class="at">GUID=</span><span class="st">&quot;8vcea&quot;</span>,<span class="at">Access_Token=</span>RPCB_private_access,<span class="at">file_name=</span><span class="st">&quot;Study_50_combined_qPCR_statistical_analysis.R&quot;</span>)</span></code></pre></div>
<pre><code>## [1] &quot;The file has been downloaded to your working directory as: Study_50_combined_qPCR_statistical_analysis.R&quot;</code></pre>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="id_05.html#cb33-1" tabindex="-1"></a><span class="fu">source</span>(<span class="st">&quot;Study_50_combined_qPCR_statistical_analysis.R&quot;</span>)</span></code></pre></div>
<pre><code>## [1] &quot;The file has been downloaded to your working directory as: Study_50_ddCT_3.17.16.R&quot;
## [1] &quot;The file has been downloaded to your working directory as: Study_50_Ct_Values_3.17.16.csv&quot;
## [1] &quot;The file has been downloaded to your working directory as: Study_50_ddCT_4.25.16.R&quot;
## [1] &quot;The file has been downloaded to your working directory as: Study_50_Ct_Values_4.25.16.csv&quot;</code></pre>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="id_05.html#cb35-1" tabindex="-1"></a><span class="co"># make data.frame of the combined data</span></span>
<span id="cb35-2"><a href="id_05.html#cb35-2" tabindex="-1"></a>log_all_total <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(log_all_3<span class="fl">.17</span><span class="sc">$</span>FoldChange, log_all_4<span class="fl">.25</span><span class="sc">$</span>FoldChange, log_all_3<span class="fl">.17</span><span class="sc">$</span>ID)</span>
<span id="cb35-3"><a href="id_05.html#cb35-3" tabindex="-1"></a><span class="fu">colnames</span>(log_all_total) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;FoldChange_3.17&quot;</span>, <span class="st">&quot;FoldChange_4.25&quot;</span>, <span class="st">&quot;ID&quot;</span>)</span>
<span id="cb35-4"><a href="id_05.html#cb35-4" tabindex="-1"></a></span>
<span id="cb35-5"><a href="id_05.html#cb35-5" tabindex="-1"></a><span class="co"># calculate average fold change, sd, and se</span></span>
<span id="cb35-6"><a href="id_05.html#cb35-6" tabindex="-1"></a>avg <span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb35-7"><a href="id_05.html#cb35-7" tabindex="-1"></a>stdev<span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb35-8"><a href="id_05.html#cb35-8" tabindex="-1"></a>sder <span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb35-9"><a href="id_05.html#cb35-9" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">40</span>) {</span>
<span id="cb35-10"><a href="id_05.html#cb35-10" tabindex="-1"></a>  avg[i] <span class="ot">&lt;-</span> (log_all_total[i,<span class="dv">1</span>]<span class="sc">+</span>log_all_total[i,<span class="dv">2</span>])<span class="sc">/</span><span class="dv">2</span></span>
<span id="cb35-11"><a href="id_05.html#cb35-11" tabindex="-1"></a>  stdev[i] <span class="ot">&lt;-</span> <span class="fu">sd</span>(<span class="fu">c</span>(log_all_total[i,<span class="dv">1</span>],log_all_total[i,<span class="dv">2</span>]))</span>
<span id="cb35-12"><a href="id_05.html#cb35-12" tabindex="-1"></a>  sder[i] <span class="ot">&lt;-</span> <span class="fu">std.error</span>(<span class="fu">c</span>(log_all_total[i,<span class="dv">1</span>],log_all_total[i,<span class="dv">2</span>]))</span>
<span id="cb35-13"><a href="id_05.html#cb35-13" tabindex="-1"></a>}</span>
<span id="cb35-14"><a href="id_05.html#cb35-14" tabindex="-1"></a></span>
<span id="cb35-15"><a href="id_05.html#cb35-15" tabindex="-1"></a><span class="co"># assign column names</span></span>
<span id="cb35-16"><a href="id_05.html#cb35-16" tabindex="-1"></a>log_all_total<span class="sc">$</span>meanFoldChange <span class="ot">&lt;-</span> avg</span>
<span id="cb35-17"><a href="id_05.html#cb35-17" tabindex="-1"></a>log_all_total<span class="sc">$</span>sd <span class="ot">&lt;-</span> stdev</span>
<span id="cb35-18"><a href="id_05.html#cb35-18" tabindex="-1"></a>log_all_total<span class="sc">$</span>se <span class="ot">&lt;-</span> sder</span>
<span id="cb35-19"><a href="id_05.html#cb35-19" tabindex="-1"></a></span>
<span id="cb35-20"><a href="id_05.html#cb35-20" tabindex="-1"></a><span class="co"># subset on samples with detectable F. nucleatum in both CRC and adjacent normal tissues (Ct below 35)</span></span>
<span id="cb35-21"><a href="id_05.html#cb35-21" tabindex="-1"></a>exp_plot <span class="ot">&lt;-</span> log_all_total[<span class="fu">which</span>(log_all_total<span class="sc">$</span>ID <span class="sc">%in%</span></span>
<span id="cb35-22"><a href="id_05.html#cb35-22" tabindex="-1"></a>                                  <span class="fu">intersect</span>(<span class="fu">gsub</span>(<span class="st">&quot;-.*&quot;</span>,<span class="st">&quot;&quot;</span>,adjnorm_Ct_35),<span class="fu">gsub</span>(<span class="st">&quot;-.*&quot;</span>,<span class="st">&quot;&quot;</span>,tumor_Ct_35))),]</span>
<span id="cb35-23"><a href="id_05.html#cb35-23" tabindex="-1"></a></span>
<span id="cb35-24"><a href="id_05.html#cb35-24" tabindex="-1"></a><span class="co"># Draw plot for fold gene expression change (2-ddCt)</span></span>
<span id="cb35-25"><a href="id_05.html#cb35-25" tabindex="-1"></a><span class="co"># Figure 1</span></span>
<span id="cb35-26"><a href="id_05.html#cb35-26" tabindex="-1"></a>Figure1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(exp_plot, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(ID,meanFoldChange), <span class="at">y =</span> meanFoldChange)) <span class="sc">+</span></span>
<span id="cb35-27"><a href="id_05.html#cb35-27" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>, <span class="at">colour =</span> <span class="st">&quot;white&quot;</span>, <span class="at">fill =</span> <span class="st">&quot;black&quot;</span>, <span class="at">width=</span><span class="fl">0.95</span>) <span class="sc">+</span></span>
<span id="cb35-28"><a href="id_05.html#cb35-28" tabindex="-1"></a>  <span class="fu">geom_errorbar</span>(<span class="at">position=</span><span class="fu">position_dodge</span>(.<span class="dv">9</span>), <span class="at">width=</span>.<span class="dv">25</span>, <span class="at">size =</span> <span class="fl">0.75</span>, <span class="fu">aes</span>(<span class="at">ymin=</span>meanFoldChange<span class="sc">-</span>se, <span class="at">ymax=</span>meanFoldChange<span class="sc">+</span>se), <span class="at">color =</span> <span class="st">&quot;grey&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-29"><a href="id_05.html#cb35-29" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">trans =</span> <span class="st">&quot;log10&quot;</span>,</span>
<span id="cb35-30"><a href="id_05.html#cb35-30" tabindex="-1"></a>                     <span class="at">breaks =</span> <span class="fu">c</span>(<span class="fl">0.1</span>,<span class="dv">1</span>,<span class="dv">10</span>,<span class="dv">100</span>),</span>
<span id="cb35-31"><a href="id_05.html#cb35-31" tabindex="-1"></a>                     <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">&quot;0.10&quot;</span>,<span class="st">&quot;1.00&quot;</span>,<span class="st">&quot;10.00&quot;</span>,<span class="st">&quot;100.00&quot;</span>),</span>
<span id="cb35-32"><a href="id_05.html#cb35-32" tabindex="-1"></a>                     <span class="at">limits =</span> <span class="fu">c</span>(<span class="fl">0.1</span>,<span class="dv">100</span>),</span>
<span id="cb35-33"><a href="id_05.html#cb35-33" tabindex="-1"></a>                     <span class="at">expand =</span> <span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">0</span>)) <span class="sc">+</span></span>
<span id="cb35-34"><a href="id_05.html#cb35-34" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Patients&quot;</span>) <span class="sc">+</span></span>
<span id="cb35-35"><a href="id_05.html#cb35-35" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="sc">~</span> <span class="fu">atop</span>(<span class="fu">paste</span>(<span class="st">&quot;Fold change in &quot;</span>, <span class="fu">italic</span>(<span class="st">&quot;F. nucleatum&quot;</span>), <span class="st">&quot; abundance&quot;</span>),</span>
<span id="cb35-36"><a href="id_05.html#cb35-36" tabindex="-1"></a>              <span class="fu">paste</span>(<span class="st">&quot;in CRC vs adjacent normal tissue&quot;</span>))) <span class="sc">+</span></span>
<span id="cb35-37"><a href="id_05.html#cb35-37" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">&quot;1&quot;</span>,<span class="st">&quot;2&quot;</span>,<span class="st">&quot;3&quot;</span>,<span class="st">&quot;4&quot;</span>,<span class="st">&quot;5&quot;</span>,<span class="st">&quot;6&quot;</span>)) <span class="sc">+</span></span>
<span id="cb35-38"><a href="id_05.html#cb35-38" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept=</span><span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb35-39"><a href="id_05.html#cb35-39" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb35-40"><a href="id_05.html#cb35-40" tabindex="-1"></a><span class="fu">theme</span>(<span class="at">panel.background =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb35-41"><a href="id_05.html#cb35-41" tabindex="-1"></a>      <span class="at">panel.grid.major.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb35-42"><a href="id_05.html#cb35-42" tabindex="-1"></a>      <span class="at">panel.grid.minor =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb35-43"><a href="id_05.html#cb35-43" tabindex="-1"></a>      <span class="at">panel.grid.major.y =</span> <span class="fu">element_line</span>(<span class="at">linewidth =</span><span class="fl">0.5</span>, <span class="at">color=</span><span class="st">&quot;grey80&quot;</span>),</span>
<span id="cb35-44"><a href="id_05.html#cb35-44" tabindex="-1"></a>      <span class="at">axis.line.x =</span> <span class="fu">element_line</span>(),</span>
<span id="cb35-45"><a href="id_05.html#cb35-45" tabindex="-1"></a>      <span class="at">axis.line.y =</span> <span class="fu">element_line</span>(),</span>
<span id="cb35-46"><a href="id_05.html#cb35-46" tabindex="-1"></a>      <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">15</span>, <span class="at">margin=</span><span class="fu">margin</span>(<span class="dv">5</span>,<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>), <span class="at">color =</span> <span class="st">&quot;black&quot;</span>),</span>
<span id="cb35-47"><a href="id_05.html#cb35-47" tabindex="-1"></a>      <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">15</span>, <span class="at">margin=</span><span class="fu">margin</span>(<span class="dv">0</span>,<span class="dv">5</span>,<span class="dv">0</span>,<span class="dv">0</span>), <span class="at">color =</span> <span class="st">&quot;black&quot;</span>),  </span>
<span id="cb35-48"><a href="id_05.html#cb35-48" tabindex="-1"></a>      <span class="at">axis.title.x =</span> <span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">20</span>,<span class="at">margin=</span><span class="fu">margin</span>(<span class="dv">20</span>,<span class="dv">0</span>,<span class="dv">0</span>,<span class="dv">0</span>)),</span>
<span id="cb35-49"><a href="id_05.html#cb35-49" tabindex="-1"></a>      <span class="at">axis.title.y =</span> <span class="fu">element_text</span>(<span class="at">size=</span><span class="dv">20</span>,<span class="at">angle=</span><span class="dv">90</span>,<span class="at">margin=</span><span class="fu">margin</span>(<span class="dv">0</span>,<span class="dv">20</span>,<span class="dv">0</span>,<span class="dv">0</span>)),</span>
<span id="cb35-50"><a href="id_05.html#cb35-50" tabindex="-1"></a>      <span class="at">axis.ticks.length =</span> <span class="fu">unit</span>(<span class="fl">0.25</span>, <span class="st">&quot;cm&quot;</span>),</span>
<span id="cb35-51"><a href="id_05.html#cb35-51" tabindex="-1"></a>      <span class="at">plot.margin =</span> <span class="fu">unit</span>(<span class="fu">c</span>(<span class="dv">1</span>,.<span class="dv">5</span>,<span class="dv">1</span>,.<span class="dv">5</span>),<span class="st">&quot;cm&quot;</span>))</span>
<span id="cb35-52"><a href="id_05.html#cb35-52" tabindex="-1"></a></span>
<span id="cb35-53"><a href="id_05.html#cb35-53" tabindex="-1"></a>title <span class="ot">&lt;-</span> <span class="fu">ggdraw</span>() <span class="sc">+</span> <span class="fu">draw_label</span>(<span class="st">&quot;Figure 1&quot;</span>, <span class="at">size=</span><span class="dv">20</span>)</span>
<span id="cb35-54"><a href="id_05.html#cb35-54" tabindex="-1"></a>Figure1 <span class="ot">&lt;-</span> <span class="fu">plot_grid</span>(title,Figure1, <span class="at">ncol =</span> <span class="dv">1</span>,<span class="at">rel_heights =</span> <span class="fu">c</span>(<span class="fl">0.1</span>,<span class="dv">1</span>))</span>
<span id="cb35-55"><a href="id_05.html#cb35-55" tabindex="-1"></a>Figure1</span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="id_05.html#cb36-1" tabindex="-1"></a><span class="co"># saves file &#39;Study_50_Figure_1.pdf&#39; locally</span></span>
<span id="cb36-2"><a href="id_05.html#cb36-2" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">&quot;Study_50_Figure_1.pdf&quot;</span>, <span class="at">dpi=</span><span class="dv">300</span>, <span class="at">height =</span> <span class="fl">6.6</span> , <span class="at">width =</span> <span class="dv">6</span>)</span></code></pre></div>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-sumnerReproducibilityReportingPractices2020" class="csl-entry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Sumner J, Haynes L, Nathan S, Hudson-Vitale C, McIntosh L (2020) <a href="https://doi.org/10.1101/2020.03.24.20042796">Reproducibility and reporting practices in <span>COVID-19</span> preprint manuscripts</a></div>
</div>
<div id="ref-tapaswiApplyingCellPainting2024" class="csl-entry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Tapaswi A, Cemalovic N, Polemi KM, Sexton JZ, Colacino JA (2024) <a href="https://doi.org/10.1101/2024.04.30.591893">Applying <span>Cell Painting</span> in <span>Non-Tumorigenic Breast Cells</span> to <span>Understand Impacts</span> of <span>Common Chemical Exposures</span></a>. 2024.04.30.591893</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="id_04.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="id_06.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true,
"toc_depth": 3
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
