<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Guerrilla Analytics | Portfolio Naresh</title>
  <meta name="description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Guerrilla Analytics | Portfolio Naresh" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  <meta name="github-repo" content="NareshdeBruijn14/Naresh_portfolio" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Guerrilla Analytics | Portfolio Naresh" />
  
  <meta name="twitter:description" content="Portfolio assignment for Data Science for Biology course at the University of Applied Science Utrecht" />
  

<meta name="author" content="Naresh de Bruijn" />


<meta name="date" content="2024-05-26" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="id_02.html"/>
<link rel="next" href="id_04.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/core-js-2.5.3/shim.min.js"></script>
<script src="libs/react-18.2.0/react.min.js"></script>
<script src="libs/react-18.2.0/react-dom.min.js"></script>
<script src="libs/reactwidget-1.0.0/react-tools.js"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="libs/reactable-binding-0.4.4/reactable.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="layout.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="toc"><h2>Table of Contents</h2></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Home Page</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#introduction"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#table-of-contents"><i class="fa fa-check"></i>Table of Contents</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_00.html"><a href="id_00.html"><i class="fa fa-check"></i>Setup</a>
<ul>
<li class="chapter" data-level="" data-path="id_00.html"><a href="id_00.html#packages"><i class="fa fa-check"></i>Packages</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html"><i class="fa fa-check"></i>Resum√©</a>
<ul>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#personal-details"><i class="fa fa-check"></i>Personal details</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#allow-me-to-introduce-myself-briefly"><i class="fa fa-check"></i>Allow me to introduce myself briefly</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#internship-major-project"><i class="fa fa-check"></i>Internship / (major) project</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#relevant-coursework"><i class="fa fa-check"></i>Relevant Coursework</a></li>
<li class="chapter" data-level="" data-path="id_01.html"><a href="id_01.html#experiences"><i class="fa fa-check"></i>Experiences</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html"><i class="fa fa-check"></i>Free schedule plan</a>
<ul>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#introduction-1"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#goals"><i class="fa fa-check"></i>Goals</a></li>
<li class="chapter" data-level="" data-path="id_02.html"><a href="id_02.html#plan"><i class="fa fa-check"></i>Plan</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_03.html"><a href="id_03.html"><i class="fa fa-check"></i>Guerrilla Analytics</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html"><i class="fa fa-check"></i>C.elegans plate experiment</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#introduction-2"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#import-and-inspect-dataset"><i class="fa fa-check"></i>Import and inspect dataset</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatter-plot"><i class="fa fa-check"></i>Scatter plot</a>
<ul>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#inspection"><i class="fa fa-check"></i>Inspection</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatterplot-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Scatterplot CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#analysis-plan"><i class="fa fa-check"></i>Analysis plan</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#normalising-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Normalising CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#scatterplot-normalised-ce_liq_flow_062_tidydata"><i class="fa fa-check"></i>Scatterplot normalised CE_LIQ_FLOW_062_Tidydata</a></li>
<li class="chapter" data-level="" data-path="id_04.html"><a href="id_04.html#conclusion"><i class="fa fa-check"></i>Conclusion</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html"><i class="fa fa-check"></i>Reproducible Research</a>
<ul>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#open-peer-review"><i class="fa fa-check"></i>Open peer review</a>
<ul>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#introduction-3"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#article-overview"><i class="fa fa-check"></i>Article overview</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#peer-review"><i class="fa fa-check"></i>Peer review</a></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#conclusion-1"><i class="fa fa-check"></i>Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_05.html"><a href="id_05.html#reproducible-analysis"><i class="fa fa-check"></i>Reproducible analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_06.html"><a href="id_06.html"><i class="fa fa-check"></i>R package</a>
<ul>
<li class="chapter" data-level="" data-path="id_06.html"><a href="id_06.html#examples"><i class="fa fa-check"></i>Examples</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html"><i class="fa fa-check"></i>Projecticum: Pediatric Cancer</a>
<ul>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#introduction-4"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#cfdna"><i class="fa fa-check"></i>cfDNA</a></li>
<li class="chapter" data-level="" data-path="id_07.html"><a href="id_07.html#r-shiny-app"><i class="fa fa-check"></i>R shiny app</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html"><i class="fa fa-check"></i>Parameterisation of COVID-19 Datasets</a>
<ul>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#introduction-5"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#parameters"><i class="fa fa-check"></i>Parameters</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#visualisation"><i class="fa fa-check"></i>Visualisation</a>
<ul>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#covid-19-cases"><i class="fa fa-check"></i>Covid-19 cases</a></li>
<li class="chapter" data-level="" data-path="id_08.html"><a href="id_08.html#covid-19-deaths"><i class="fa fa-check"></i>Covid-19 deaths</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html"><i class="fa fa-check"></i>BS-Seq</a>
<ul>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#introduction-6"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#bisulfite-conversion"><i class="fa fa-check"></i>Bisulfite conversion</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#bismark"><i class="fa fa-check"></i>Bismark</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#data"><i class="fa fa-check"></i>Data</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#genome-preparation"><i class="fa fa-check"></i>Genome preparation</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#alignment"><i class="fa fa-check"></i>Alignment</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#deduplication"><i class="fa fa-check"></i>Deduplication</a></li>
<li class="chapter" data-level="" data-path="id_09.html"><a href="id_09.html#methylation-extraction"><i class="fa fa-check"></i>Methylation extraction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="id_20.html"><a href="id_20.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="1" data-path="dsfb2_portfolio.html"><a href="dsfb2_portfolio.html"><i class="fa fa-check"></i><b>1</b> DSFB2_portfolio</a></li>
<li class="chapter" data-level="2" data-path="section.html"><a href="section.html"><i class="fa fa-check"></i><b>2</b> </a></li>
<li class="chapter" data-level="3" data-path="direction-tree.html"><a href="direction-tree.html"><i class="fa fa-check"></i><b>3</b> Direction tree</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio Naresh</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class="hero-image"></div>

<link href="layout.css" rel="stylesheet">
<div class="hero-image-container">
  <img class= "hero-image" src="documents/images/DNA_helix_blue.png">
</div>

<div id="id_03" class="section level1 unnumbered">
<h1>Guerrilla Analytics</h1>
<div id="example-guerrilla-analytics-daur-ii" class="section level4 unnumbered">
<h4>Example Guerrilla Analytics DAUR-II</h4>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="id_03.html#cb2-1" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(here<span class="sc">::</span><span class="fu">here</span>(</span>
<span id="cb2-2"><a href="id_03.html#cb2-2" tabindex="-1"></a>  <span class="st">&quot;documents&quot;</span>,</span>
<span id="cb2-3"><a href="id_03.html#cb2-3" tabindex="-1"></a>  <span class="st">&quot;images&quot;</span>,</span>
<span id="cb2-4"><a href="id_03.html#cb2-4" tabindex="-1"></a>  <span class="st">&quot;Guerrilla_Analytics_metagenomics.png&quot;</span></span>
<span id="cb2-5"><a href="id_03.html#cb2-5" tabindex="-1"></a>))</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:unnamed-chunk-1"></span>
<img src="documents/images/Guerrilla_Analytics_metagenomics.png" alt="Guerrilla Analytics tree structure of DAUR-II." width="702" />
<p class="caption">
Figure 0.1: Guerrilla Analytics tree structure of DAUR-II.
</p>
</div>
</div>
<div id="root-of-projecticum-peadiatric-cancer-project" class="section level4 unnumbered">
<h4>Root of Projecticum: Peadiatric Cancer project</h4>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="id_03.html#cb3-1" tabindex="-1"></a><span class="co"># print root of Projecticum project repo</span></span>
<span id="cb3-2"><a href="id_03.html#cb3-2" tabindex="-1"></a>fs<span class="sc">::</span><span class="fu">dir_tree</span>(</span>
<span id="cb3-3"><a href="id_03.html#cb3-3" tabindex="-1"></a>  here<span class="sc">::</span><span class="fu">here</span>(<span class="st">&quot;../DSFB2_boys/&quot;</span>), </span>
<span id="cb3-4"><a href="id_03.html#cb3-4" tabindex="-1"></a>             <span class="at">recurse =</span> T, </span>
<span id="cb3-5"><a href="id_03.html#cb3-5" tabindex="-1"></a>             <span class="at">regexp =</span> <span class="st">&quot;^.gitignore$|^[^_|(.git)]&quot;</span>,</span>
<span id="cb3-6"><a href="id_03.html#cb3-6" tabindex="-1"></a>             <span class="at">all =</span>F)</span></code></pre></div>
<pre><code>## /Users/naresh/Documents/08_DSFB/DSFB2_portfolio/../DSFB2_boys/
## ‚îú‚îÄ‚îÄ Conflict4_2
## ‚îú‚îÄ‚îÄ DSFB2_Project.Rproj
## ‚îú‚îÄ‚îÄ DSFB2_boys.Rproj
## ‚îú‚îÄ‚îÄ LICENSE
## ‚îú‚îÄ‚îÄ README.html
## ‚îú‚îÄ‚îÄ README.md
## ‚îú‚îÄ‚îÄ code
## ‚îÇ   ‚îú‚îÄ‚îÄ Global.R
## ‚îÇ   ‚îú‚îÄ‚îÄ server.R
## ‚îÇ   ‚îî‚îÄ‚îÄ ui.R
## ‚îú‚îÄ‚îÄ data
## ‚îÇ   ‚îî‚îÄ‚îÄ liquid_biopsies_data
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000AAA_HHUBM000CAA_HHUCZ000AAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000AAA_HHUCZ000AAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000ACA_HHUBM000ABA_HHUCZ000ACA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000ACA_HHUCZ000ACA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000ADA_HHUBM000CAA_HHUCZ000ADA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000ADA_HHUCZ000ADA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000BAA_HHUBM000CAA_HHUCZ000BAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000BAA_HHUCZ000BAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000EAA_HHUBM000FAA_HHUCZ000EAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000EAA_HHUCZ000EAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000GAA_HHUBM000IAA_HHUCZ000GAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000GAA_HHUCZ000GAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000HAA_HHUBM000IAA_HHUCZ000HAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000HAA_HHUCZ000HAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000KAA_HHUBM000LAA_HHUCZ000KAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000KAA_HHUCZ000KAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000YAA_HHUBM000ABA_HHUCZ000YAA_WXS.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000YAA_HHUCZ000YAA_WXS.tumor.cnv.vcf
## ‚îÇ       ‚îú‚îÄ‚îÄ HHUBM000ZAA_HHUBM000ABA_HHUCZ000ZAA_WXS.vcf
## ‚îÇ       ‚îî‚îÄ‚îÄ HHUBM000ZAA_HHUCZ000ZAA_WXS.tumor.cnv.vcf
## ‚îú‚îÄ‚îÄ documents
## ‚îÇ   ‚îú‚îÄ‚îÄ Project_description.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ bibliography.bib
## ‚îÇ   ‚îú‚îÄ‚îÄ images
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ logo.png
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ maxima_logo.png
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ screenshots
## ‚îÇ   ‚îÇ       ‚îî‚îÄ‚îÄ dir_tree_project.png
## ‚îÇ   ‚îú‚îÄ‚îÄ logo.tex
## ‚îÇ   ‚îú‚îÄ‚îÄ project_introduction.Rmd
## ‚îÇ   ‚îî‚îÄ‚îÄ project_introduction.pdf
## ‚îú‚îÄ‚îÄ raw_data
## ‚îÇ   ‚îú‚îÄ‚îÄ Project_description.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ   ‚îú‚îÄ‚îÄ code
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Global.R
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ server.R
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ ui.R
## ‚îÇ   ‚îú‚îÄ‚îÄ liquid_biopsies_data
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000AAA_HHUBM000CAA_HHUCZ000AAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000AAA_HHUCZ000AAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000ACA_HHUBM000ABA_HHUCZ000ACA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000ACA_HHUCZ000ACA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000ADA_HHUBM000CAA_HHUCZ000ADA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000ADA_HHUCZ000ADA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000BAA_HHUBM000CAA_HHUCZ000BAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000BAA_HHUCZ000BAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000EAA_HHUBM000FAA_HHUCZ000EAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000EAA_HHUCZ000EAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000GAA_HHUBM000IAA_HHUCZ000GAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000GAA_HHUCZ000GAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000HAA_HHUBM000IAA_HHUCZ000HAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000HAA_HHUCZ000HAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000KAA_HHUBM000LAA_HHUCZ000KAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000KAA_HHUCZ000KAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000YAA_HHUBM000ABA_HHUCZ000YAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000YAA_HHUCZ000YAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ HHUBM000ZAA_HHUBM000ABA_HHUCZ000ZAA_WXS.vcf
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ HHUBM000ZAA_HHUCZ000ZAA_WXS.tumor.cnv.vcf.gz
## ‚îÇ   ‚îú‚îÄ‚îÄ readme.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ resources
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ 20220628_gene_list.txt
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Genetic_aberrations_Wilms_tumor_20230328.csv
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ RMS_gene_list.tsv
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ fake_ascat_results.tsv
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ list_CN_add.txt
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ maf_cfDNA_list.rds
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ maf_tumor_list.rds
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ trecode_info_byPMCID_inSampleInfo.tsv
## ‚îÇ   ‚îî‚îÄ‚îÄ scripts
## ‚îÇ       ‚îî‚îÄ‚îÄ functions.R
## ‚îú‚îÄ‚îÄ resources
## ‚îÇ   ‚îú‚îÄ‚îÄ 20220628_gene_list.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ Genetic_aberrations_Wilms_tumor_20230328.csv
## ‚îÇ   ‚îú‚îÄ‚îÄ RMS_gene_list.tsv
## ‚îÇ   ‚îú‚îÄ‚îÄ fake_ascat_results.tsv
## ‚îÇ   ‚îú‚îÄ‚îÄ list_CN_add.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ maf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ maf_cfDNA_list.rds
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ maf_tumor_list.rds
## ‚îÇ   ‚îî‚îÄ‚îÄ trecode_info_byPMCID_inSampleInfo.tsv
## ‚îî‚îÄ‚îÄ scripts
##     ‚îú‚îÄ‚îÄ decompress_cnv_files.R
##     ‚îî‚îÄ‚îÄ functions.R</code></pre>
</div>
<div id="root-of-workflow-portfolio-project" class="section level4 unnumbered">
<h4>Root of Workflow portfolio project</h4>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="id_03.html#cb5-1" tabindex="-1"></a><span class="co"># print root of workflows portfolio project</span></span>
<span id="cb5-2"><a href="id_03.html#cb5-2" tabindex="-1"></a>fs<span class="sc">::</span><span class="fu">dir_tree</span>(<span class="st">&quot;.&quot;</span>, </span>
<span id="cb5-3"><a href="id_03.html#cb5-3" tabindex="-1"></a>             <span class="at">recurse =</span> T, </span>
<span id="cb5-4"><a href="id_03.html#cb5-4" tabindex="-1"></a>             <span class="at">regexp =</span> <span class="st">&quot;^</span><span class="sc">\\</span><span class="st">.gitignore$|^[^_|(.git)]&quot;</span>,</span>
<span id="cb5-5"><a href="id_03.html#cb5-5" tabindex="-1"></a>             <span class="at">all =</span> F)</span></code></pre></div>
<pre><code>## .
## ‚îú‚îÄ‚îÄ 00_Setup.Rmd
## ‚îú‚îÄ‚îÄ 01_CV.Rmd
## ‚îú‚îÄ‚îÄ 02_Free_schedule_plan.Rmd
## ‚îú‚îÄ‚îÄ 03_Guerrilla_Analytics.Rmd
## ‚îú‚îÄ‚îÄ 04_Celegans_plate_experiment.Rmd
## ‚îú‚îÄ‚îÄ 05_Reproducible_Research.Rmd
## ‚îú‚îÄ‚îÄ 06_R_package.Rmd
## ‚îú‚îÄ‚îÄ 07_Projecticum_introduction.Rmd
## ‚îú‚îÄ‚îÄ 08_Covid-19_params.Rmd
## ‚îú‚îÄ‚îÄ 09_BS-Seq.Rmd
## ‚îú‚îÄ‚îÄ 20_references.Rmd
## ‚îú‚îÄ‚îÄ DSFB2_portfolio.Rproj
## ‚îú‚îÄ‚îÄ LICENSE
## ‚îú‚îÄ‚îÄ README.Rmd
## ‚îú‚îÄ‚îÄ README.md
## ‚îú‚îÄ‚îÄ code
## ‚îÇ   ‚îú‚îÄ‚îÄ README.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ Study_50
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_Ct_Values_3.17.16.csv
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_Ct_Values_4.25.16.csv
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_Figure_1.pdf
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_combined_qPCR_statistical_analysis.R
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_ddCT_3.17.16.R
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Study_50_ddCT_4.25.16.R
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ download.OSF.file.R
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ test
## ‚îÇ   ‚îî‚îÄ‚îÄ fastqc.sh
## ‚îú‚îÄ‚îÄ data
## ‚îÇ   ‚îú‚îÄ‚îÄ CE.LIQ.FLOW.062_Tidydata.xlsx
## ‚îÇ   ‚îú‚îÄ‚îÄ README.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ fastq
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ SRR29020037.fastq.gz
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ SRR9834204.fastq.gz
## ‚îÇ   ‚îú‚îÄ‚îÄ fastqc_output
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ SRR9834204_fastqc.html
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ SRR9834204_fastqc.zip
## ‚îÇ   ‚îú‚îÄ‚îÄ genomes
## ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Homo_sapiens.GRCh38.dna.alt.fa.gz
## ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ human_GRCh38_dataset
## ‚îÇ   ‚îÇ       ‚îú‚îÄ‚îÄ README.md
## ‚îÇ   ‚îÇ       ‚îî‚îÄ‚îÄ ncbi_dataset
## ‚îÇ   ‚îÇ           ‚îú‚îÄ‚îÄ data
## ‚îÇ   ‚îÇ           ‚îÇ   ‚îú‚îÄ‚îÄ GCF_000001405.40
## ‚îÇ   ‚îÇ           ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ GRCh38.p14.fna.gz
## ‚îÇ   ‚îÇ           ‚îÇ   ‚îú‚îÄ‚îÄ assembly_data_report.jsonl
## ‚îÇ   ‚îÇ           ‚îÇ   ‚îî‚îÄ‚îÄ dataset_catalog.json
## ‚îÇ   ‚îÇ           ‚îî‚îÄ‚îÄ fetch.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ species.csv
## ‚îÇ   ‚îî‚îÄ‚îÄ tools
## ‚îÇ       ‚îú‚îÄ‚îÄ Bismark
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ Bismark_alignment_modes.pdf
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ CHANGELOG.md
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ NOMe_filtering
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ _config.yml
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bam2nuc
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark2bedGraph
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark2report
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark2summary
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark_genome_preparation
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bismark_methylation_extractor
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ copy_bismark_files_for_release.pl
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ coverage2cytosine
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ deduplicate_bismark
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ docs
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ alignment.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ concordance.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ coverage_report.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ deduplication.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ filter_nonconverted_reads.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ genome_preparation.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ library_types.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ methylation_extraction.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ processing_report.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ summary_report.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ faq
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ context_change.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ conversion_efficiency.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ low_mapping.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ single_cell_pbat.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ images
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Bismark_Bowtie2_HISAT2.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ PBAT_SE_M-bias.jpg
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ PE_M-bias.jpg
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bioinformatics_logo.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bioinformatics_logo.svg
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_report.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_report.txt
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ methylation_concordance.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ nucleocoverage.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ summary.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ installation.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ options
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ alignment.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ genome_preparation.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ methylation_extraction.md
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ quick_reference.md
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ filter_non_conversion
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ license.txt
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ merge_arbitrary_coverage_files.py
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ merge_coverage_files_ARGV.py
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ methylation_consistency
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ mkdocs.yml
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ plotly
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ Bismark_Alignment_Stats_Summary.png
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bioinf.logo
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark.logo
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_bt2_PE_report.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_RRBS.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_WGBS.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_single_cells.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ bismark_summary_single_cells.txt
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ plot.ly
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ plotly_template.tpl
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ test_data.fastq
## ‚îÇ       ‚îÇ   ‚îî‚îÄ‚îÄ test_files
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ NC_010473.fa.gz
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ test_R1.fastq.gz
## ‚îÇ       ‚îÇ       ‚îî‚îÄ‚îÄ test_R2.fastq.gz
## ‚îÇ       ‚îú‚îÄ‚îÄ bowtie2
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ AUTHORS
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ BOWTIE2_VERSION
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ LICENSE
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ MANUAL
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ MANUAL.markdown
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ NEWS
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ README.md
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ TUTORIAL
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-align-l
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-align-l-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-align-s
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-align-s-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-build
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-build-l
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-build-l-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-build-s
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-build-s-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-inspect
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-inspect-l
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-inspect-l-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-inspect-s
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ bowtie2-inspect-s-debug
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ doc
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ README
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ manual.html
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ style.css
## ‚îÇ       ‚îÇ   ‚îú‚îÄ‚îÄ example
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ index
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ lambda_virus.1.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ lambda_virus.2.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ lambda_virus.3.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ lambda_virus.4.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ lambda_virus.rev.1.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ lambda_virus.rev.2.bt2
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ reads
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ combined_reads.bam
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ longreads.fq
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ reads_1.fq
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îú‚îÄ‚îÄ reads_2.fq
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ simulate.pl
## ‚îÇ       ‚îÇ   ‚îÇ   ‚îî‚îÄ‚îÄ reference
## ‚îÇ       ‚îÇ   ‚îÇ       ‚îî‚îÄ‚îÄ lambda_virus.fa
## ‚îÇ       ‚îÇ   ‚îî‚îÄ‚îÄ scripts
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ bowtie2-hbb.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ convert_quals.pl
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ gen_2b_occ_lookup.pl
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ gen_occ_lookup.pl
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ gen_solqual_lookup.pl
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ infer_fraglen.pl
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_a_thaliana_tair.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_b_taurus_UMD3.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_c_elegans.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_canFam2.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_d_melanogaster.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_e_coli.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_h_sapiens_ncbi36.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_h_sapiens_ncbi37.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_hg18.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_hg19.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_m_musculus_ncbi37.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_mm10.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_mm9.sh
## ‚îÇ       ‚îÇ       ‚îú‚îÄ‚îÄ make_rn4.sh
## ‚îÇ       ‚îÇ       ‚îî‚îÄ‚îÄ make_s_cerevisiae.sh
## ‚îÇ       ‚îú‚îÄ‚îÄ dataformat
## ‚îÇ       ‚îî‚îÄ‚îÄ datasets
## ‚îú‚îÄ‚îÄ documents
## ‚îÇ   ‚îú‚îÄ‚îÄ README.txt
## ‚îÇ   ‚îú‚îÄ‚îÄ bibliography.bib
## ‚îÇ   ‚îî‚îÄ‚îÄ images
## ‚îÇ       ‚îú‚îÄ‚îÄ DNA_helix_blue.png
## ‚îÇ       ‚îú‚îÄ‚îÄ DNA_helix_logo.png
## ‚îÇ       ‚îú‚îÄ‚îÄ Guerrilla_Analytics_metagenomics.png
## ‚îÇ       ‚îú‚îÄ‚îÄ bs-seq_method.jpeg
## ‚îÇ       ‚îî‚îÄ‚îÄ resume_pic.png
## ‚îú‚îÄ‚îÄ hero-image.html
## ‚îú‚îÄ‚îÄ layout.css
## ‚îú‚îÄ‚îÄ nohup.out
## ‚îî‚îÄ‚îÄ raw_data
##     ‚îú‚îÄ‚îÄ Bismark-0.24.2.tar
##     ‚îú‚îÄ‚îÄ CE.LIQ.FLOW.062_Tidydata.xlsx
##     ‚îú‚îÄ‚îÄ README.txt
##     ‚îú‚îÄ‚îÄ fastq
##     ‚îÇ   ‚îú‚îÄ‚îÄ SRR29020037.fastq.gz
##     ‚îÇ   ‚îî‚îÄ‚îÄ SRR9834204.fastq.gz
##     ‚îú‚îÄ‚îÄ human_GRCh38_dataset.zip
##     ‚îî‚îÄ‚îÄ species.csv</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="id_02.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="id_04.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true,
"toc_depth": 3
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
